crates/language/src/language.rs

```diff
@@ -337,7 +336,7 @@ pub trait LspAdapter: 'static + Send + Sync {
     }
 
     /// Returns initialization options that are going to be sent to a LSP server as a part of [`lsp_types::InitializeParams`]
-    async fn initialization_options(&self) -> Option<Value> {
+    fn initialization_options(&self) -> Option<Value> {
```

Comment:
Is this a safe change? I'm worried some language server will need async initialization for some file operation. Could be an irrelevant concern, just thought I might bring it up.

Reply:
Sounds good, thanks for clarifying.